jwst/tweakreg/tweakreg_step.py

```diff
@@ -277,34 +283,36 @@ def process(self, input):
         # absolute alignment to the reference catalog
         # can (and does) occur after alignment between groups
         if align_to_abs_refcat:
-            try:
-                ref_image = images[0]
-                correctors = \
-                    twk.absolute_align(correctors, self.abs_refcat,
-                                       ref_wcs=ref_image.meta.wcs,
-                                       ref_wcsinfo=ref_image.meta.wcsinfo.instance,
-                                       epoch=Time(ref_image.meta.observation.date).decimalyear,
-                                       abs_minobj=self.abs_minobj,
-                                       abs_fitgeometry=self.abs_fitgeometry,
-                                       abs_nclip=self.abs_nclip,
-                                       abs_sigma=self.abs_sigma,
-                                       abs_searchrad=self.abs_searchrad,
-                                       abs_use2dhist=self.abs_use2dhist,
-                                       abs_separation=self.abs_separation,
-                                       abs_tolerance=self.abs_tolerance,
-                                       save_abs_catalog=self.save_abs_catalog,
-                                       abs_catalog_output_dir=self.output_dir,
-                                            )
-
-            except twk.TweakregError as e:
-                self.log.warning(str(e))
-                for model in images:
-                    model.meta.cal_step.tweakreg = "SKIPPED"
-                return images 
-
-        if local_align_failed and not align_to_abs_refcat:    
-            for model in images:
-                record_step_status(model, "tweakreg", success=False)
+            with images:
+                try:
+                    ref_image = images.borrow(0)
```

Comment:
```suggestion
                ref_image = images.borrow(0)
                try:
```
Since we don't expect `borrow` to raise `TweakregError`